.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       matrix:
         include:
           - os: ubuntu-latest
-            lib: libindy_credx.so
+            lib: anoncreds.so
```

Comment:
This should likely be libanoncreds.so, and libanoncreds.dylib below, just to match platform standards.

Reply:
Correct. Not sure how I wrote it this way :)